### PR TITLE
Add infinity scroll

### PR DESCRIPTION
**Pull request by mikelgarciaurbina**
I added an infinity scroll to the dashboard and transactions page. I also removed the "APARTMENT" icon because is not exist.

### CODE REVIEW

File: src/modules/groupTxsByDate.js [hunk 1]
```diff
@@ -10,10 +10,10 @@ export const groupTxsByDate = (txs = []) => {
     if (date !== txDate) {
       date = txDate;
       dateIndex = query.length;
-      query.push({ timestamp: tx.timestamp, txs: [] });
+      query.push({ timestamp: tx.timestamp, data: [] });
```
Comment (soyjavi) [inline src/modules/groupTxsByDate.js:13]: why use a different name?

File: src/screens/Main/Dashboard/Dashboard.jsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import { Action, Input, Screen, ScrollView } from '@satoshi-ltd/nano-design';
 import PropTypes from 'prop-types';
 import React, { useEffect, useState } from 'react';
+import { SectionList } from 'react-native';
```
Comment (soyjavi) [inline src/screens/Main/Dashboard/Dashboard.jsx]: this one should be part of nano-design.. the general idea is money don't use any `react-native` component by itself.

File: src/screens/Main/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -37,7 +39,7 @@ const Dashboard = ({ navigation: { navigate } = {} }) => {
   const sortedAccounts = queryAccounts({ accounts, query: undefined });
 
   return (
-    <Screen>
+    <Screen disableScroll style={[style.container]}>
```
Comment (soyjavi) [inline src/screens/Main/Dashboard/Dashboard.jsx]: style={style.container}

File: src/modules/groupTxsByDate.js (hunk 1 above)
Comment (mikelgarciaurbina) [reply src/modules/groupTxsByDate.js:13]: The SectionList component needs items to be called data

File: src/screens/Main/Dashboard/Dashboard.jsx (hunk 2 above)
Comment (mikelgarciaurbina) [reply src/screens/Main/Dashboard/Dashboard.jsx]: Done

File: src/screens/Main/Dashboard/Dashboard.jsx (hunk 3 above)
Comment (mikelgarciaurbina) [reply src/screens/Main/Dashboard/Dashboard.jsx]: Done

File: src/screens/Main/Dashboard/Dashboard.jsx [hunk 4]
```diff
@@ -92,10 +93,14 @@ const Dashboard = ({ navigation: { navigate } = {} }) => {
...
             />
           )}
-
-          {(querySearchTxs({ accounts, query, txs }) || lastTxs).map((item) => (
-            <GroupTransactions {...item} key={`${item.timestamp}`} currency={baseCurrency} />
-          ))}
+          <SectionList
+            dataSource={querySearchTxs({ accounts, query, txs, page }) || lastTxs}
+            keyExtractor={(item) => item.timestamp}
+            onEndReached={() => setPage(page + 1)}
```
Comment (soyjavi) [inline src/screens/Main/Dashboard/Dashboard.jsx]: events on after common properties

File: src/screens/Main/Dashboard/Dashboard.style.js [hunk 5]
```diff
@@ -4,12 +4,17 @@ export const style = StyleSheet.create({
   scrollView: {
     marginBottom: '$spaceXL',
     paddingRight: '$spaceM',
+    maxHeight: '$cardAccountSize',
```
Comment (soyjavi) [inline src/screens/Main/Dashboard/Dashboard.style.js:6]: alfabetic sort

File: src/screens/Main/Dashboard/helpers/queryLastTxs.js [hunk 6]
```diff
@@ -1,10 +1,10 @@
 import { C, groupTxsByDate } from '../../../../modules';
 
-const { CURRENCY } = C;
-export const queryLastTxs = ({ accounts = [], txs = [] }) =>
+const { CURRENCY, ITEMS_PER_PAGE } = C;
```
Comment (soyjavi) [inline src/screens/Main/Dashboard/helpers/queryLastTxs.js:3]: return

File: src/screens/Transactions/Transactions.jsx [hunk 7]
```diff
@@ -17,24 +16,18 @@ const {
...
 const Transactions = ({ route: { params: { account: { hash } } = {} } = {}, navigation = {} }) => {
-  const scrollview = useRef(null);
   const { accounts = [], settings: { baseCurrency } = {} } = useStore();
 
-  const { height } = useWindowDimensions();
-
   const [dataSource, setDataSource] = useState({});
-  const [scrollQuery, setScrollQuery] = useState(false);
   const [txs, setTxs] = useState([]);
+  const [page, setPage] = useState(2);
```
Comment (soyjavi) [inline src/screens/Transactions/Transactions.jsx:23]: why `2`?

File: src/screens/Transactions/Transactions.jsx [hunk 8]
```diff
@@ -45,55 +38,50 @@ const Transactions = ({ route: { params: { account: { hash } } = {} } = {}, navi
...
+            renderItem={({ item }) => <GroupTransactionsItem currency={baseCurrency} {...item} />}
+            renderSectionHeader={({ section }) => <GroupTransactions {...section} />}
           />
-        )}
-      </ScrollView>
+        </>
+      ) : (
+        <Banner
+          align="center"
+          // ! TODO: Setup image
```
Comment (soyjavi) [inline src/screens/Transactions/Transactions.jsx]: comments

File: src/components/GroupTransactions/GroupTransactions.Item.jsx [hunk 9]
```diff
@@ -15,7 +15,15 @@ const {
   INTERNAL_TRANSFER,
 } = C;
 
-const Item = ({ category = INTERNAL_TRANSFER, currency, timestamp, title, type = EXPENSE, value = 0, ...others }) => {
+const GroupTransactionsItem = ({
```
Comment (soyjavi) [inline src/components/GroupTransactions/GroupTransactions.Item.jsx:18]: linter?

File: src/screens/Transactions/Transactions.jsx (hunk 7 above)
Comment (mikelgarciaurbina) [reply src/screens/Transactions/Transactions.jsx:23]: To have the first 40 items rendered at the first time

File: src/components/GroupTransactions/GroupTransactions.Item.jsx (hunk 9 above)
Comment (mikelgarciaurbina) [reply src/components/GroupTransactions/GroupTransactions.Item.jsx:18]: Yes, when I changed the name of the component, the linter added the returns.

**Pull request merged**